src/inc/readytorun.h

```diff
@@ -335,6 +335,9 @@ enum ReadyToRunHelper
 
     // JIT32 x86-specific exception handling
     READYTORUN_HELPER_EndCatch                  = 0x110,
+
+    // Stack probing helper
+    READYTORUN_HELPER_StackProbe                = 0x111,
```

Comment:
Is there a R2R versioning that needs to change due to this?

Reply:
I don't know.

@jkotas Can you please advise if the R2R version needs to be change when we add a new READYTORUN helper?

Reply:
We have bumped R2R version a few days ago for .NET 5, so you do need to worry about bumping it again.

Reply:
@jkotas To clarify, you say "so you do need to worry about bumping it again". Do you mean this, or do you mean "you do NOT need to worry..."? It seems like you mean NOT.

Reply:
Yes, I meant NOT. Sorry for the confusion.

Reply:
@jkotas Thank you for the reply!